Execution failed for task ':react-native-text-detector:verifyReleaseResources'

**Execution failed for task ':react-native-text-detector:verifyReleaseResources'**

I'm trying to generate the APK file and when it reaches 96%, this error, below, occurs. I could not find a solution and I need to urgently generate the update of the App. Has anyone gone through this and can point out a possible solution?

- **My versions**

Android Studio version: 3.4
Gradle version: 5.1.1
react-native-cli: 2.0.1
react-native: 0.55.4
npm: 6.9.0

- **In my android/build.gradle I have:**

dependencies {
    classpath 'com.android.tools.build:gradle:3.4.0'
    classpath 'com.google.gms:google-services:4.1.0'
    classpath 'com.bugsnag:bugsnag-android-gradle-plugin:4.+'
}

ext {
    buildToolsVersion = "27.0.3"
    minSdkVersion = 16
    compileSdkVersion = 28
    targetSdkVersion = 28
    supportLibVersion = '26.1.0'
    googlePlayServicesVersion = "16.+"
    firebaseVersion = "17.6.0"    
}

- **In my ./app/build.gradle file, I have:**

 compileSdkVersion rootProject.ext.compileSdkVersion
 buildToolsVersion '27.0.3'
 defaultConfig {
     applicationId "com.rvm"
     minSdkVersion rootProject.ext.minSdkVersion
     targetSdkVersion rootProject.ext.targetSdkVersion
     versionCode 1
     versionName "1.0"
     multiDexEnabled true
     ndk {
         abiFilters "armeabi-v7a", "x86"
     }
...
}

------------------------------------------------------------
**### The error:**

> Task :react-native-text-detector:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-text-detector:verifyReleaseResources'.
> 1 exception was raised by workers:
  com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  D:\src\RN\rn_rvm\node_modules\react-native-text-detector\android\build\intermediates\res\merged\release\values\values.xml:1785: error: resource android:attr/fontVariationSettings not found.
  D:\src\RN\rn_rvm\node_modules\react-native-text-detector\android\build\intermediates\res\merged\release\values\values.xml:1786: error: resource android:attr/ttcIndex not found.
  error: failed linking references.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -17,12 +17,12 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 27)
-    buildToolsVersion safeExtGet('buildToolsVersion', '26.0.3')
+    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
 
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
-        targetSdkVersion safeExtGet('targetSdkVersion', 27)
+        targetSdkVersion safeExtGet('targetSdkVersion', 28)
         versionCode 1
         versionName "1.0"
     }

